.github/workflows/ubuntu-rvm.yml

```diff
@@ -0,0 +1,47 @@
+name: ubuntu-rvm
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby: [ 'ruby-head' ]
+        test_tool: [ "rubygems", "bundler" ]
+        include:
+          - ruby: ruby-head
+            test_tool: bundler
+            rgv: ..
+            bdv: master
+    steps:
+    - uses: actions/checkout@master
+    - run: git submodule update -i
+    - name: Fixed world writable dirs
+      run: |
+        chmod go-w $HOME
+        sudo chmod -R go-w /usr/share
+        sudo chmod -R go-w /opt
+    - name: Set up RVM
+      run: |
+        curl -sSL https://get.rvm.io | bash
+    - name: Set up Ruby
+      run: |
+        source $HOME/.rvm/scripts/rvm
+        rvm install ${{ matrix.ruby }} --binary
+        rvm --default use ${{ matrix.ruby }}
+    - name: Install dependencies
+      run: |
+        source $HOME/.rvm/scripts/rvm
+        util/ci.sh before_script
+      env:
+        TEST_TOOL: ${{ matrix.test_tool }}
+    - name: Run test
+      run: |
+        source $HOME/.rvm/scripts/rvm
+        util/ci.sh script
+      continue-on-error: true
+      env:
+        TEST_TOOL: ${{ matrix.test_tool }}
+        RGV: ${{ matrix.rgv }}
+        BDV: ${{ matrix.bdv }}
```

Comment:
The reason for this separate workflow is that there's currently no way to use `ruby-head` in github actions without using `rvm`. Right?

Reply:
At present, ruby-head & JRuby are not 'natively' available on any Actions platform/OS.